xhr.bs

```diff
@@ -851,7 +836,8 @@ return <a>this</a>'s <a>cross-origin credentials</a>.
     </ol>
     <!-- upload complete flag can never be set here I hope -->
 
-    <p>To <a>process response</a> for <var>response</var>, run these steps:
+   <li>
+    <p>Let <var>processResponse</var>, given a <var>response</var>, be these steps:
```

Comment:
Step 2 below has a typo "fo" instead of "for"

Also, most steps after step 1 seem to use "this's response", but step 2 uses "response".